doc/internal/man3/evp_keymgmt_util_export_to_provider.pod

```diff
@@ -27,16 +30,24 @@ via a B<EVP_KEYMGMT> interface, if this hasn't already been done.
 It maintains a cache of provider key references in I<pk> to keep track
 of all provider side keys.
 
-To export a legacy key, use L<evp_pkey_make_provided(3)> instead, as
-this function deals purely with provider side keys and will not care
-to look at any legacy key.
+To export a legacy key, use L<evp_pkey_export_to_provider(3)> instead,
+as this function deals purely with provider side keys and will not
+care to look at any legacy key.
 
-evp_keymgmt_util_clear_pkey_cache() can be used to explicitly clear
-the cache of provider key references.
+evp_keymgmt_util_find_operation_cache_index() finds the location if
```

Comment:
finds the index of 
I<keymgmt> in I<pk>'s cache of provider sides keys used for operations.

<Maybe this bit goes in the returns section??>
If I<keymgmt> is set to NULL then it returns the index of the first empty slot in the cache.

Reply:
I rephrased a bit, both here and in RETURN VALUES.